pkg/cortex/cortex.go

```diff
@@ -301,6 +302,12 @@ func New(cfg Config) (*Cortex, error) {
 		os.Exit(0)
 	}
 
+	// TODO: Find a better place for this
```

Comment:
I think it needs to be set as soon as possible, but I am not too sure if this is a good place. Thoughts?

Reply:
**thought:** I don't think it's a big deal if this stays here. The alternative is making a small module and setting it as a dependency for the `Server` module. Having a module for such a small snippet may be overkill, but it doesn't necessarily have to be scoped only for this config, it could be a generic `init` module. The Auth config logic below could also be moved there for instance.

Reply:
Ok will keep it like that for now